1.segment-data/README.md

```diff
@@ -101,8 +94,8 @@ We used an external harddrive and therefore needed to use specific paths.
 bash 1.segment-data.sh
 ```
 
-**Note:** With GPU enabled, our estimated run time was about **100 hours** to segment all Cell Health data.
-The total data for all segmentation data are about **20 GB**.
+**Note:** With GPU enabled, our estimated run time was about **50 hours** to segment all Cell Health data.
```

Comment:
Interesting that it look less time but it is more data!

Reply:
Yep, the mask data was already being derived for nuclei before, and now we don't get cytoplasm segmentations. Unfortunately, saving these mask images makes a lot more data.